arrocke/gloss-translation
# feat: show progress in the translation view

## What has changed
- The `TranslationView `displays a book progress bar directly below the `TranslationToolbar`

## Connected Issues

closes #411 

## QA Steps
1. Go to a verse and hover over the progress bar. Confirm that the approved words to total words count is accurate. 
2. Approve a gloss for a word and hover over the progress bar again. Confirm that the approved words count has gone up by one and the total words count has remained the same. 
3. Link two words and give them an approved gloss. Confirm that the approved words count has gone up by two and the total words count has remained the same.
4. In VSCode navigate to `TranslationToolbar`. Manually replace the value of the local variable `percentageFull` variable to 25%.  Reload the app.
5. Confirm that the progress text is now displayed within the teal section of the progress bar.
## Post-Deployment
